modules/docextract/lib/refextract_kbs.py

```diff
@@ -194,7 +193,6 @@ def institute_num_pattern_to_regex(pattern):
     """
     simple_replacements = [
                             ('9',    r'\d'),
-                            ('9+',   r'\d+'),
```

Comment:
The whole pattern substitution logic was broken:
* first the pattern was escaped, but `+` was not the whitelist, so this would lead to `9+` -> `9\+`
* the replacements were tried sequentially, so the `9` rule was applied before the `9+` rule anyway

By not escaping patterns, we only need substitutions for the "words", not the quantifiers.